src/rest-client.ts

```diff
@@ -376,8 +376,8 @@ export class RestClient {
     return this.requestWrapper.get(`spot_margin/market_info${suffix}`);
   }
 
-  getBorrowHistory(): GenericAPIResponse {
-    return this.requestWrapper.get(`spot_margin/borrow_history`);
+  getBorrowHistory(params: TimeRange): GenericAPIResponse {
```

Comment:
Damn just after merging I realised something - the params are optional, so it should be `params?: TimeRange`. I'll do a quick fix with the release PR.

Reply:
Ah yes indeed, pagination is always optional
So in this case it could be called without params...

Reply:
Yes..totally missed it. If all properties in the interface are optional then the param should also be optional. I've updated it before publishing for 1.1.7, see #69